src/renderers/dom/test/__tests__/ReactTestUtils-test.js

```diff
@@ -844,4 +844,20 @@ describe('ReactTestUtils', () => {
       );
     });
   });
+
+  it('should call setState with the correct parameters', () => {
```

Comment:
Let's change this to `should call setState callback with no arguments` (and others too)